COPYING

```diff
@@ -16,14 +16,14 @@ licensed as MIT:
 
     THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
     IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
-    THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
-    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+    FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL
+    THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES, OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT, OR OTHERWISE, ARISING
+    FROM, OUT OF, OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
     DEALINGS IN THE SOFTWARE.
 
 However, when linked together to form a Linux kernel module, the resulting Linux
-kernel module is dual licensed as MIT/GPLv2.
+the kernel module is dual licensed as MIT/GPLv2.
```

Comment:
This suggestion is in error.

Currently
_the_ resulting Linux kernel module is dual licensed as MIT/GPLv2.

vs

Your suggestion
_the_ resulting Linux _the_ kernel module is dual licensed as MIT/GPLv2.



I do agree with the assessment that it's not a good idea to proof read software licenses in projects that use them. Nice and consistent is key.